db/migrate/20220726202412_create_doc_sentencing_counties.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDocSentencingCounties < ActiveRecord::Migration[6.0]
+  def change
+    create_table :doc_sentencing_counties do |t|
+      t.string :name
```

Comment:
I think it makes sense to match the backend constraint to the model constraint

```suggestion
      t.string :name, null: false
```

Reply:
should I rollback and edit  the file here or make a new db:migrate? either way I assume we will pair on this on friday correct?

Reply:
Also Fixed the error  by adding sent_counties back in,  I think I accidentally deleted it beforehand.